packages/charts/src/components/BulletChart/BulletChart.tsx

```diff
@@ -460,7 +469,17 @@ const BulletChart: FC<BulletChartProps> = forwardRef((props: BulletChartProps, r
               type="monotone"
               dataKey={element.accessor}
               {...chartElementProps}
-            />
+            >
+              {dataset.map((data, i) => {
+                return (
+                  <Cell
+                    key={i}
+                    fill={getCellColors(element, data, index)}
+                    stroke={getCellColors(element, data, index)}
```

Comment:
By having this code the default color `black` for the comparison line is now ignored. is this intended?